Book/php7/build_system/building_php.rst

```diff
@@ -78,7 +78,10 @@ have the first three installed by default):
 * ``gcc`` or some other compiler suite.
 * ``libc-dev``, which provides the C standard library, including headers.
 * ``make``, which is the build-management tool PHP uses.
-* ``autoconf`` (2.64 or higher), which is used to generate the ``configure`` script.
+* ``autoconf``, which is used to generate the ``configure`` script.
+  * 2.59 or higher (for PHP 7.0-7.1)
+  * 2.64 or higher (for PHP 7.2)
+  * 2.68 or higher (for PHP 7.3)
```

Comment:
This probably needs to be indented by one more space.

Reply:
Yes, ah, sorry... I've missed the RST format. Another line separation was required.